lib/jekyll-sitemap.rb

```diff
@@ -23,9 +29,9 @@ def generate(site)
       end
     end
 
-    # Array of all non-jekyll site files with an HTML extension
-    def html_files
-      @site.static_files.select { |file| File.extname(file.relative_path) == ".html" }
+    # Array of all non-jekyll site files with given extensions
+    def exts_files (exts=[])
+      @site.static_files.select { |file| exts.include? File.extname(file.relative_path) }
```

Comment:
Instead of `File.extname(file.relative_path)`, we should use `file.extname`. This would allow the class to cache this value to save time.